components/Buttons.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { MouseEventHandler } from 'react';
+
+interface Props {
```

Comment:
Rename the interface using more specific naming, i.e. `ButtonProps`. Maybe you could also inherit the default React button props too.